lib/matplotlib/axes/_axes.py

```diff
@@ -322,7 +322,7 @@ def inset_axes(self, bounds, *, transform=None, zorder=5, **kwargs):
             Lower-left corner of inset Axes, and its width and height.
 
         transform : `.Transform`
-            Defaults to `ax.transAxes`, i.e. the units of *rect* are in
+            Defaults to ``ax.transAxes``, i.e. the units of *rect* are in
```

Comment:
We should have a documentation reference for this one way or the other. To be checked: why don't we have this in our api reference.

Same for some other cases below.

Reply:
I think the problem here is that `ax` is an object, so maybe one should refer to the class instead.

Reply:
The deeper problem is that `transAxes` is a simple attribute, which is not documented, and even if it was, we currently cannot reference it (c.f. `dataLim`).

Let's go for literal for the time being. I've opened #22897 to track this.